src/onboarding/Start.js

```diff
@@ -207,10 +207,19 @@ class StartContent extends React.PureComponent {
     if (!hasAccount) {
       return <ActionInfo>Please unlock MetaMask.</ActionInfo>
     }
+    if (network.type === 'unknown') {
+      return (
+        <ActionInfo>
+          You are using an unsupported chain. Please switch to rinkeby or a
+          local test environment instead.
```

Comment:
We should probably only mention “or local test environment instead” when `process.env.NODE_ENV !== 'production'`.

Reply:
Network is definitely better :+1: